code/game/machinery/doors/airlock.dm

```diff
@@ -1186,14 +1186,15 @@
 
 		var/time_to_open = 5
 		if(hasPower() && !prying_so_hard)
-			time_to_open = 50
-			playsound(src, 'sound/machines/airlock_alien_prying.ogg', 100, TRUE) //is it aliens or just the CE being a dick?
-			prying_so_hard = TRUE
-			if(do_after(user, time_to_open, TRUE, src))
-				open(2)
-				if(density && !open(2))
-					to_chat(user, "<span class='warning'>Despite your attempts, [src] refuses to open.</span>")
-			prying_so_hard = FALSE
+			if (I.tool_behaviour == TOOL_CROWBAR) //we need another check, futureproofing for if/when bettertools actually completely replaces the old jaws
+				time_to_open = 50
```

Comment:
Can you define this variable here instead?
It being defined a few lines above to 5 then changing it when it is only being used in this section it kind of "shitcode" so you'd help our codebase be a better place.

Reply:
what do you mean? All I did was modify the existing code

Reply:
this is intended

Reply:
I mean make the line here var/time_to_open = 50 then delete the var/time_to_open = 5 a few lines above

Reply:
no this is inteded

Reply:
that's how the code works.

Reply:
The time is = 5 if the door is unpowered
Time is set to 50 if it's powered

Reply:
Alright.